### PR TITLE
Updates to work with Ansible 2

### PR DESCRIPTION
**Pull request by andrewkroh**
Tested with Ansible 2.3.1.0 which is what [beats-ci](https://beats-ci.elastic.co/job/elastic+beats-tester+master/) is using.

### CODE REVIEW

File: roles/test-win-winlogbeat-file/tasks/main.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@
       - "registry_stat.stat.size > 0"
 
 - name: Verify registry contents
-  script: run_script.ps1 -script "'cat {{ winlogbeat_data_dir }}\\.winlogbeat.yml'"
+  win_shell: "cat {{ winlogbeat_data_dir }}\\.winlogbeat.yml"
```
Comment (tsg) [inline roles/test-win-winlogbeat-file/tasks/main.yml]: Sounds like the `run_script.ps1` could be deleted in this case?

Comment (tsg) [inline roles/test-win-winlogbeat-file/tasks/main.yml]: Ah, i see it's still used for installing/uninstalling. Nevermind, then.

Comment (andrewkroh) [inline roles/test-win-winlogbeat-file/tasks/main.yml]: Yeah, I think there was a problem with the quoting on the output in the two cases that I modified. I tried to make the least amount of changes possible get it working. There is moer cleanup we can do and more deprecation warnings to fix for Ansible 2.

**Comment by andrewkroh** [comment]
It errors on the Jenkins b/c that's the old Jenkins. It should pass on the new Jenkins once it's merged to master. We will add a PR job to the new Jenkins for beats-tester soon.

**Pull request merged**